WPMedia/fusion-news-theme-blocks
# THEMES-835 Hero block image resizer updates

## Description

The Hero - Arc Block was developed prior to Reziser V2 being available. Due to this we need to go back and update the Hero - Arc Block to use Resizer V2.

## Jira Ticket

- [THEMES-835](https://arcpublishing.atlassian.net/browse/THEMES-835)

## Acceptance Criteria

Hero - Arc Block is using Resizer V2

## Test Steps

1. Checkout this blocks repo branch `git checkout THEMES-835-Resizer-Hero-block`
2. Run fusion repo commerce branch with linked blocks `npx fusion start -f -l @wpmedia/hero-block`
3. Configure content to display
4. Inspect on image to verify the image uses v2 resizer

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-835]: https://arcpublishing.atlassian.net/browse/THEMES-835?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
